scripts/thrust_model_v2/Static/EmaxStatic/graph_thrust_current.m

```diff
@@ -0,0 +1,76 @@
+% Levi Burner 2-10-18

```

Comment:
Why are there 5 different versions of this that look very similar?  Are there actually meaningful differences?